### PR TITLE
Wyswietlanie wynikow z backendu

### PR DESCRIPTION
**Pull request by krzysztofwolny**
Some styling and basic components has been added for purpose of displaying dummy data from backend

### CODE REVIEW

File: client/src/components/RepoList/RepoListItem/RepoListItem.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { RepoListItemWrapper, RepoListItemItems } from './RepoListItem.styles';
+
+const RepoListItem = ({ username, repositories_count }) => {
+    return(
+        <RepoListItemWrapper>
+            <RepoListItemItems>Github nick: {username}</RepoListItemItems>
+            <RepoListItemItems>Number of repositories: {repositories_count}</RepoListItemItems>
```
Comment (hannakozak) [inline client/src/components/RepoList/RepoListItem/RepoListItem.js:8]: Why not add parameters like: language, number of stars, name of repositories?

File: client/src/dummyData/usersGithubUsers.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    {
+        username: 'user9',
+        repositories_count: 3
+    },
+    {
+        username: 'user10',
+        repositories_count: 69
+    },
+];
```
Comment (hannakozak) [inline client/src/dummyData/usersGithubUsers.js:38]: Like i mentioned before, i would add all parameters, we are going to receive from backend later.

**Review by hannakozak — APPROVED** [review]
Good job. We will work on styling later on.

File: client/src/components/RepoList/RepoListItem/RepoListItem.js (hunk 1 above)
Comment (krzysztofwolny) [reply client/src/components/RepoList/RepoListItem/RepoListItem.js:8]: I would be better for sure, but if I correctly understood the backend/app/models.py file this is what we currently get from the endpoint

File: client/src/dummyData/usersGithubUsers.js (hunk 2 above)
Comment (krzysztofwolny) [reply client/src/dummyData/usersGithubUsers.js:38]: On next sync meeting we are going (I hope so)  to create our API template. Then we will know exactly what we are going to recive

**Review by orzeech — APPROVED** [review]

**Pull request merged**